lib/actions/auth.coffee

```diff
@@ -27,7 +27,7 @@ exports.login	=
 
 		- Credentials: using email/password and 2FA.
 
-		- Token: using the authentication token from the preferences page.
+		- Token: using a session token or API key (experimental) from the preferences page.
```

Comment:
Those docs are actually talking about the old api keys, which I think are deprecated anyway... For now we're not announcing this yet, since support is still pretty unstable (as you've found), but we will want to go through all of that and update everything later once that's sorted.

Reply:
Oh my bad :(

Reply:
No no, sorry, I should've been clearer!